desc/compute/_geometry.py

```diff
@@ -275,25 +282,22 @@ def _A_of_r(params, transforms, profiles, data, **kwargs):
     label="S",
     units="m^{2}",
     units_long="square meters",
-    description="Surface area of outermost flux surface",
+    description="Surface area of outermost flux surface, scaled by max(ρ)⁻¹",
```

Comment:
Do we want this?

Reply:
Yes, but I also changed the compute function so that we scale by max rho. Do we want to do that?